index.html

```diff
@@ -54,5 +55,156 @@ <h2 class="start-btn">Start collaboration</h2>
         alt=""
       />
     </div>
-  </body>
+    <section class="project-header">
+      <div class="header-div">
+        <h1 class="header-2">Projects</h1>
+
+        <div class="project-img">
+          <img src="images/first.png" class="image-1" alt="" />
+        </div>
+      </div>
+      <div class="project-details">
```

Comment:
- [ ] Nice work you have done with the projects section, kindly add some padding between the image and the title to give it some space, so it aligns with what is in the Figma Template.
- [ ] Kindly align the see this project `button to the left and give it some padding so it matchess what we have in the Figma Template.
- [ ] Kindly change the `font` and `font-size` of the text in the tags so it aligns with what is on the Figma Template see below!

## Figma Template
![card-n-1](https://user-images.githubusercontent.com/48371156/166115546-20391876-c1bd-4b7a-a0d6-8a300f47d86e.png)

## Your Design
![card-n-2](https://user-images.githubusercontent.com/48371156/166115487-c29a73fe-5b61-4fc4-906c-973fc52c62e0.png)

Reply:
thanks, i hope I met the requirements now, I'm mentally and physically drained, worked for two days straight without sleep just to complete these projects and meet up with the deadline. thanks once more